features/schemas/ua_security_status.json

```diff
@@ -48,7 +66,7 @@
         }
       }
     },
-    "packages": {
+    "updates": {
```

Comment:
This is a major breaking change unfortunately. We would either need to be sure no-one is using this command before we make a change like this, Or we'd need to implement schema versioning for the command

Reply:
There are, right now, more changes requested which will probably break the structure a couple times, until we get the response to a "stable" state...
We do have a `schema_version` field though. Should we just update that?
Or do you think we should by now already be thinking of retrocompatibility?

Reply:
Because we removed the `--beta` flag in 27.7, I think we need to be backwards compatible from now on.
So any major change in schema like changing key names will require a new separate schema and we'll need to support both. We can default to the old schema and opt-in to the new with a `--version=2` flag, for example.